lib/hexpm_web/router.ex

```diff
@@ -14,11 +14,17 @@ defmodule HexpmWeb.Router do
     plug :put_secure_browser_headers
     plug :web_user_agent
     plug :validate_url
-    plug :login
+  end
+
+  pipeline :defaults do
```

Comment:
Probably not the best name.

Reply:
Why is this moved to a separate pipeline?

Reply:
Because both plugin in this pipeline require login to be set. Login has to do a db lookup, and the same one HexpmWeb.Session does, for the authenticated area I didn't want us to do that lookup twice. In another PR I'd like re-work this area some more so that there maybe can be one pipeline. Make sense?